automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/lab/LabNotebookPage.scala

```diff
@@ -16,67 +16,82 @@ import scala.concurrent.duration._
 import scala.language.postfixOps
 import scala.util.{Failure, Success, Try}
 
-
-class LabNotebookPage(override val url: String)(override implicit val authToken: AuthToken, override implicit val webDriver: WebDriver)
-  extends LabPage with Toolbar with NotebookCell with Eventually with LazyLogging {
-
-
-  override def open(implicit webDriver: WebDriver): LabNotebookPage = {
+class LabNotebookPage(override val url: String)(implicit override val authToken: AuthToken,
+                                                implicit override val webDriver: WebDriver)
+    extends LabPage
+    with Toolbar
+    with NotebookCell
+    with Eventually
+    with LazyLogging {
+
+  override def open(implicit webDriver: WebDriver): LabNotebookPage =
     super.open.asInstanceOf[LabNotebookPage]
-  }
 
   // menu elements
 
   lazy val fileMenu: Element = {
-    findAll(cssSelector(menus)).filter { e => e.text == "File" }.toList.head
+    findAll(cssSelector(menus))
+      .filter { e =>
+        e.text == "File"
+      }
```

Comment:
Splitting this over 3 lines is overkill but I'm guessing the formatter can't be smart enough to figure out that the functional literal is pretty short.